modules/administration-guide/pages/configuring-workspace-target-namespace.adoc

```diff
@@ -0,0 +1,62 @@
+:_content-type: PROCEDURE
+:navtitle: Configuring {orch-namespace} name
+:keywords: administration guide, configuring, namespace
+:page-aliases: 
+
+[id="configuring-user-{orch-namespace}-name-for-automatic-provisioning_{context}"]
+= Configuring user {orch-namespace} name for automatic provisioning
+
+You can configure the {orch-namespace} name template that {prod-short} is using to create the required {orch-namespace} when starting a workspace.
+
+A valid {orch-namespace} name template follows this convention:
```

Comment:
```suggestion
A valid {orch-namespace} name template follows these conventions:
```
Plural needed because there are multiple conventions in the list